backend/rpc.py

```
@@ -1,29 +1,47 @@
+from helpers import json_replacer, json_reviver
 import playtime
 
+
+def rpcmethod(func: Callable):
+  args_names = func.__code__.co_varnames[:func.__code__.co_argcount]
+  @wraps(func)
+  def wrapper_func(payload: str):
+    data = json.loads(payload, object_hook=json_reviver)
+    args = itemgetter(*args_names)(data)
+    ret = func(*args) if isinstance(args, tuple) else func(args)
+    if ret: return json.dumps(ret, default=json_replacer)
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)